jobfunnel/backend/scrapers/monster.py

```diff
@@ -406,3 +406,32 @@ def _get_search_url(self, method: Optional[str] = 'get',
             raise NotImplementedError()
         else:
             raise ValueError(f'No html method {method} exists')   
+
+class MonsterScraperGEGer(MonsterMetricRadius, BaseMonsterScraper,
+                           BaseGEGerScraper):
+    """Scrapes jobs from www.monster.de
+    """
+    def _get_search_url(self, method: Optional[str] = 'get',
+                        page: int = 1) -> str:
+        """Get the monster search url from SearchTerms
+        TODO: implement fulltime/part-time portion + company search?
+        TODO: implement POST
+        NOTE: unfortunately we cannot start on any page other than 1,
+            so the jobs displayed just scrolls forever and we will see
+            all previous jobs as we go.
+        """
+        if method == 'get':
+            return (
+                'https://www.monster.{}/jobs/search/?{}q={}&where={}'
+                    '&rad={}'.format(
+                        self.config.search_config.domain,
+                        f'page={page}&' if page > 1 else '',
+                        self.query,
+                        self.config.search_config.city.replace(' ', '-'),
+                        self._convert_radius(self.config.search_config.radius)
```

Comment:
Does German's Monster allow searching by state? If so I think perhaps this is missing here.